README.md

```diff
@@ -30,12 +30,48 @@ Also:
 
 `ember-infinity` exposes 3 consumable items for your application.
 
-· **Route Mixin**
+· **infinity service**
 
 · **infinity-loader component**
 
-· **infinity service**
+· **Route Mixin** (optional)
```

Comment:
Let's mark this as deprecated

Reply:
marked as deprecated until 1.1.0.  Lmk if you think that is an ok version to remove at....